tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
We are not going to use Typescript to this project. So we need not configure this